strawberryfields/gbs/vibronic.py

```diff
@@ -145,27 +159,37 @@ def gbs_params(
     >>>     ]
     >>> )
     >>> d = np.array([0.2254, 0.1469, 1.5599, -0.3784, 0.4553, -0.3439, 0.0618])
-    >>> p = gbs_params(w, wp, Ud, d)
+    >>> T = 0.0
+    >>> p = gbs_params(w, wp, Ud, d, T)
 
     Args:
         w (array): normal mode frequencies of the electronic ground state (:math:`\text{cm}^{-1}`)
```

Comment:
```suggestion
        w (array): normal mode frequencies of the electronic ground state (:math:`\mbox{cm}^{-1}`)
```
I realised this doesn't render correctly